internal/datastore/crdb/crdb.go

```diff
@@ -307,6 +308,17 @@ func (cds *crdbDatastore) HeadRevision(ctx context.Context) (datastore.Revision,
 	return hlcNow, err
 }
 
+func (cds *crdbDatastore) Features(ctx context.Context) (*datastore.Features, error) {
+	var features datastore.Features
+	err := cds.pool.BeginTxFunc(ctx, pgx.TxOptions{AccessMode: pgx.ReadOnly}, func(tx pgx.Tx) error {
```

Comment:
I don't think you need to use BeginTxFunc here. You can just use `QueryRow` on the pool and it will do the right thing.